### DISCUSSION TITLE
Cosmos chain onboarding proposal - Soarchain

### DISCUSSION THREAD

**Discussion by Alperen-Dere** [wormhole-foundation/wormhole#4060] [Gateway]
### Website

https://www.soarchain.com/

### Github

https://github.com/soar-robotics

### Blog

https://www.blog.soarchain.com/

### Documentation

https://docs.soarchain.com/

### Community stats

Followers on Twitter : 32.000
Followers on Telegram : 2.800
Users on Discord : 15.000

### Short Summary

​Soarchain is a Layer 1 built on Cosmos SDK network that enables a token-incentivized data and connectivity infrastructure for mobility. With its native smart contracts, Soarchain enables decentralized applications specific to mobility, creating a dAPP ecosystem on cars, leading to a revolution of more efficient, cost-effective, and eco-friendly transportation.

### Why should Wormhole add this chain?

LBP: liquidity boostraping pool: Fjord
Soarchain wants to create a wrapped token using wormhole

Guardians are not required to run a full node for soarchain - connectivity will be via a Gateway IBC connection.

### Technology and Features

- Runner Nodes:
These are specialized nodes that handle a significant portion of data processing tasks, such as data management and forming Merkle trees. Runner Nodes help reduce the load on the primary network, addressing scalability issues and enabling Soarchain to handle increased transaction volumes.

- zk-SNARKs:
Utilized for robust data verification while maintaining confidentiality. zk-SNARKs enable vehicles to prove the authenticity and integrity of their data without revealing the actual content, ensuring privacy and security within the Soarchain network.

- Proof-of-Availability (PoAv):
A verification method to ensure that data from devices in the network is accurately received and recorded. This involves selective verification and constructing Merkle trees within Soarchain's verification layer.

- Interoperability and Cross-Industry Collaboration:
Soarchain collaborates with various blockchain networks (e.g., Fetch.ai, Akash, Dymension, and Jackal Protocol) and AI companies (e.g., Archetype AI, Nukl.AI) to enhance interoperability and provide robust infrastructure support

- Interchain stack:
Layer-1 Decentralized Physical Infrastructure Network (DePIN):
Soarchain's foundation is built on a Layer-1 DePIN, designed to manage and integrate data from various sources, particularly focusing on mobility and edge applications.

CometBFT:
Utilizes Byzantine Fault Tolerant (BFT) State Machine Replication (SMR) to ensure robust, secure, and consistent replication of the blockchain application across multiple machines. This helps protect against Sybil attacks and double-spending.

Cosmos SDK:
The modular architecture of Cosmos SDK facilitates the development of custom modules for specific use cases, such as Proof-of-Availability and Data Provision Request modules. It allows Soarchain to manage computational resources efficiently and adjust the cost of computation as needed.

CosmWasm:
Provides a flexible and upgradable smart contract platform, enabling Soarchain to adapt and upgrade its network according to the evolving needs of its users. This ensures that Soarchain remains a cutting-edge solution for decentralized mobility networks.

### Native token details

Max Supply: 9 billion soar 
soar token is the native token of soarchain for staking and fees.

### Technical requirements for running a node

None - soarchain will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers).

### IBC Relaying Strategy

Soarchain foundation will start IBC relayers initially.

**Comment by Pumckuu** [comment]
ну да, хорошая идея.
